source/views/calendar/types.js

```diff
@@ -15,6 +15,7 @@ export type GoogleEventType = {
 type EmbeddedEventDetailType = {type: 'google', data: GoogleEventType}
 
 export type EventType = {
+  title?: string,
```

Comment:
What is title? Isn't "summary" the title?

Reply:
https://github.com/StoDevX/AAO-React-Native/blob/ea4871ac31ef6b67028f2d8151e3f1bafdb8ca2e/source/views/calendar/clean-event.js#L7

cleanEvent should return a CleanedEventType, which things should be updated to use. 

And this "title" property should be removed from EventType because it's not in EventType anymore.

Reply:
Ah. Didn't understand it became a new type. I'll fix that up. Thanks.